CHANGELOG.rst

```diff
@@ -1,5 +1,50 @@
 Changelog
 =========
+1.7.0 (2020-03-02)
+------------------
+MLflow 1.7.0 includes several major features and improvements, and some notable breaking changes:
+
+MLflow support for Python 2 is now deprecated and will be dropped in a future release. At that
+point, existing Python 2 workflows that use MLflow will continue to work without modification, but
+Python 2 users will no longer get access to the latest MLflow features and bugfixes. We recommend
+that you upgrade to Python 3 - see  https://docs.python.org/3/howto/pyporting.html for a migration
+guide.
+
+Breaking changes to Model Registry REST APIs:
+
+Model Registry REST APIs are updated to be more consistent with the other MLflow APIs. With this
+release Model Registry APIs are no longer beta and intended to be stable until the next major
+version. Python and Java client APIs for Model Registry are backward compatible and have been
+updated to use new REST APIs. However, when using MLflow client with a server using updated REST
+endpoints, you will need to upgrade to new client version without changing any user code.
```

Comment:
However, when using MLflow client with a server using updated REST
endpoints, you will need to upgrade to new client version without changing any user code.

-> However, when using an MLflow client with a server using updated REST
endpoints, you won't need to change any code but will need to upgrade to a new client version.